docker-compose-rule-core/src/main/java/com/palantir/docker/compose/execution/DefaultDockerCompose.java

```diff
@@ -195,6 +193,10 @@ public String config() throws IOException, InterruptedException {
     @Override
     public boolean writeLogs(String container, OutputStream output) throws IOException {
         try {
+            Awaitility.await()
+                    .pollInterval(50, TimeUnit.MILLISECONDS)
+                    .forever()
```

Comment:
Use atMost with a timeout (i.e. COMMAND_TIMEOUT) rather than leaving it unbounded